addons/website/static/src/xml/website.pageProperties.xml

```diff
@@ -130,12 +130,10 @@
                 <div class="form-group row">
                     <label class="control-label col-md-5" for="date_publish">Publishing Date</label>
                     <div class="col-md-7">
-                        <div class='input-group date' id='date_publish_container'>
-                            <input type='text' class="form-control" name="date_publish" id="date_publish" />
-                            <div class="input-group-append">
-                                <span class="input-group-text">
-                                    <span class="fa fa-calendar" role="img" aria-label="Publish date" title="Publish date"></span>
-                                </span>
+                        <div class="input-group date" id="date_publish_container" data-target-input="nearest">
+                            <input type="text" class="form-control datetimepicker-input" data-target="#date_publish_container"/>
```

Comment:
You can't remove the id `date_publish`..
It won't save anymore, did you try to save the page property dialog? It's not saving anymore..

https://github.com/odoo-dev/odoo/blame/6185eb9ac71afba79a3815a2de197c7cb76ea21c/addons/website/static/src/js/menu/content.js#L208

Reply:
I guess I forgot to do a -u website before testing the last modification, really sorry about that.

Reply:
No `-u` is required for static files...